core/tests/tests/String.mint

```diff
@@ -137,3 +137,32 @@ suite "String.trim" {
     String.trim("   \n\n\r\r   ") == ""
   }
 }
+
+suite "String.isNotEmpty" {
+  test "it returns true for non-empty string" {
+    String.isNotEmpty("a") == true
+  }
+
+  test "it returns false for fully empty string" {
+    String.isNotEmpty("") == false
+  }
+
+  test "it returns false for empty string (whitespace)" {
+    String.isNotEmpty(
+      "
+      ") == false
+  }
```

Comment:
```suggestion
  test "it returns true for empty string (whitespace)" {
    String.isNotEmpty("  ") == true
  }
```

Reply:
I've added `String.isBlank` and `String.isNotBlank` :smile: :+1: